.prow.yaml

```diff
@@ -0,0 +1,31 @@
+presubmits:
+  - name: kustomize-build
+    decorate: true
+    max_concurrency: 1
+    run_if_changed: ".*yaml"
+    skip_report: false
+    context: aicoe-ci/prow/kustomize-build
+    spec:
+      containers:
+        - image: quay.io/operate-first/opf-toolbox:v0.3.1
+          command:
+            - "test-kustomize-build"
```

Comment:
is this  to run from the image ? if so, isn't the file kept at /usr/local/bin/ ?

Reply:
/usr/local/bin is already in the $PATH right? so we can run like this from anywhere?

Reply:
yes, it's run from the image, the script is in `/usr/local/bin` which is in `PATH`, though I've discovered some permissions issue, I'll fix that asap so it's callable directly

Reply:
i saw the permission issue as well in my local, that why i asked :)

Reply:
Thanks @tumido